bika/lims/content/analysisrequest.py

```diff
@@ -61,13 +61,6 @@
 
 
 schema = BikaSchema.copy() + Schema((
-    # The ID assigned to the client's request by the lab
-    ComputedField(
-        'RequestID',
-        searchable=True,
-        expression="here.getId()",
-        widget=ComputedWidget(visible=False),
-    ),
```

Comment:
Just in case, for backwards compatibility, better if you add the function below:
```python
@deprecated('Flagged on 170328. Use setId(id) instead, but only as a last resort')
def setRequestID(self, id):
    """ Delegates to setId(id) function """
    self.setId(id)
```